.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
This PR does not solve this: https://extrahorizon.atlassian.net/browse/SVCS-2678 , so extracted it to a different ticket.